src/Serilog/Core/LevelOverrideMap.cs

```diff
@@ -68,7 +68,7 @@ public LevelOverride(string context, LoggingLevelSwitch levelSwitch)
     {
         foreach (var levelOverride in _overrides)
         {
-            if (context.StartsWith(levelOverride.Context) &&
+            if (context.StartsWith(levelOverride.Context, StringComparison.CurrentCultureIgnoreCase) &&
```

Comment:
would OrdinalIgnoreCase work? Introducing a culture dependency is not something I'd do lightly

Reply:
That was a reasonable choice.
I guess the [perf of switching to ordinal](https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/String.Comparison.cs,23804789ea4c9c0e) would be a good thing too, if we assume that the values, being programmatic identifiers, should hopefully not require culture-sensitivity.

Reply:
We steer clear of the default/current culture whenever possible, because it can lead to surprising and unexpected deployment-time behavior changes. Most culture-specific API use in Serilog either relies on either ordinal/"invariant" culture operations, if the task is programmatic and wouldn't generally be expected to vary in behavior based on the _user's_ culture, or it accepts an `IFormatProvider` or equivalent.

In this case, the programmer's culture and the eventual deployment environment culture might not bear any relationship to each other at all (e.g. the program and its config files might be written in Australia, but run on a machine in Germany), so ordinal/invariant comparisons seem like the best choice.

Reply:
@nblumhardt  thanks for the explanation

Reply:
>Australia, but run on a machine in Germany

Missed opportunity to say "in Austria" :P